app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -406,4 +407,9 @@ class PatientRepository @Inject constructor(
         }
         .andThen(reportsRepository.deleteReportsFile().toCompletable())
   }
+
+  fun recentPatients(facilityUuid: UUID): Observable<List<RecentPatient>> =
+      database.recentPatientDao()
+          .recentPatients(facilityUuid, limit = 10)
```

Comment:
The `limit` here is again presentation logic, and should be decided by the user of this method, and not this class itself. This is something that we would want to change via remote config, so it should injected via a config object.